pkg/install/v1alpha1/env.go

```diff
@@ -35,4 +35,12 @@ const (
 	//
 	// Default is "true"
 	CreateDefaultStorageConfig menv.ENVKey = "OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG"
+
+	// InstallCRD is the environment
+	// variable that flags if maya apiserver should install the CRDs
+	// As the installation moves towards helm 3, the responsibility of installing
+	// CRDs can be pushed to helm.
+	//
+	// Default is "true"
+	InstallCRD menv.ENVKey = "OPENEBS_IO_INSTALL_CRD"
```

Comment:
can we name it like `InstallV1Alpha1CRDs` ?

Reply:
Not sure I understand the reason? Will there be a case that we will allow installing other types of CRDs?

Reply:
Ignore the above comment, i just got confused with ENV name going to expose to user